pr_check.sh

```diff
@@ -1,12 +1,24 @@
 #!/bin/bash
 
-set -exv
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+export APP_NAME="compliance"  # name of app-sre "application" folder this component lives in
+export COMPONENT_NAME="ingress"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
```

Comment:
Shouldn't this be "compliance" ? 

it looks like being used [here](https://github.com/RedHatInsights/bonfire/blob/master/cicd/cji_smoke_test.sh#L29)

Reply:
Updated. Thanks!

Reply:
Updated. Thanks!